README.md

```diff
@@ -166,6 +138,26 @@ into other projects or build by itself.
 | `ENABLE_NC`             |   `OFF` | Enable installing TLS-enabled nc(1)                                                                             |
 | `OPENSSLDIR`            |   Blank | Set the default openssl directory.  Can be specified from command line using <br>```-DOPENSSLDIR=<dirname>```   |
 
+## Build information for specific systems
+
+### HP-UX (11i)
+
+Set the UNIX_STD environment variable to `2003` before running `configure`
+in order to build with the HP C/aC++ compiler. See the "standards(5)" man
+page for more details.
+
+```sh
+export UNIX_STD=2003
+./configure
+make
+```
+
+### mingw-w64 - Windows
+
+LibreSSL builds against relatively recent versions of Mingw-w64, not to be
+confused with the original mingw.org project. Mingw-w64 3.2 or later
```

Comment:
"Mingw-w64" appears to often be stylised as MinGW-w64 (see https://www.mingw-w64.org/).
Additionally, I think linking to the MinGW-w64 website may be useful.

```suggestion
LibreSSL builds against relatively recent versions of [MinGW-w64](https://www.mingw-w64.org/), not to be
confused with the original mingw.org project. MinGW-w64 3.2 or later
```